l10n_br_sale/models/sale_order.py

```diff
@@ -182,7 +182,17 @@ def _prepare_invoice(self):
         result = super()._prepare_invoice()
         # O caso Brasil se caracteriza por ter a Operação Fiscal
         if self.fiscal_operation_id:
-            result.update(self._prepare_br_fiscal_dict())
+            # TODO: O metodo prepare_br_fiscal_dict retorna o partner_id
+            #  principal e não está considerando os casos do partner_invoice_id
+            #  diferente do partner, deveria?
```

Comment:
@mbcosta entendo que vamos ter pedidos desta forma. Neste caso entendo que os dados de endereço para a fatura deve ser do partner_invocie_id (se houver dado) e os dados fiscais deve pegar do parceiro comercial

    @api.depends('is_company', 'parent_id.commercial_partner_id')
    def _compute_commercial_partner(self):
        for partner in self:
            if partner.is_company or not partner.parent_id:
                partner.commercial_partner_id = partner
            else:
                partner.commercial_partner_id = partner.parent_id.commercial_partner_id
                
O Neto está trabalhando com uma proposta para o fiscal tbm.. https://github.com/OCA/l10n-brazil/pull/2520

Reply:
Estou dando uma olhada no modo nativo..

Reply:
No modo nativo o VAT nem é editável e entendo que o CNPJ/CPF tbm nao deve ser

![image](https://github.com/OCA/l10n-brazil/assets/3595132/5a85ac66-3275-416a-8b72-f79e3797b71d)